### DISCUSSION TITLE
Configuration of custom CSV table

### DISCUSSION THREAD

**Discussion by fir999** [WinMerge/winmerge#1767] [Ideas]
I compared two CSV tabes with custom delimiter-separeted values. It was tedious to figure out how to configure WinMerge:

1. The delimiter has to be set to ";" (obvious step).
2. The custom file pattern must be set to "*.csv" (not obvious, because CSV files are already recognized as tables).
3. The CSV file pattern "*.csv" has to be deleted (not obvious at all).

My suggestion is to offer only one file pattern in the configuration dialog so that the user can easily see which configuration is used.

**Comment by sdottaka** [comment]
It is different from your suggestion, but in version 2.16.29 released today, you can specify the delimiter of the CSV file as follows.

![image](https://user-images.githubusercontent.com/98126/226630887-8ab7109e-a068-4f59-b908-115bab42eb05.png)

**Reply by fir999** [reply]
Many thanks for the quick adjustment!
